json-logic/test/beard_definition_test.js

```diff
@@ -0,0 +1,39 @@
+const jsonLogic = require("./logic");
+const beardExamples = require("../../shared/example_beards.json");
+const beardRules = require("../beard_definition.json");
+
+const beardData = beardExamples.map(function(beard){
+  let data = {}
+  if(beard.facial_hair_over_5mm === "true") {
```

Comment:
This is a little bit of a hack, but comes up against something interesting about the data...

All that it actually needs for the decision is whether it's over 5mm - but if we just take a 'facial hair over 5mm' variable, that rule is effectively delegated to the caller, and we don't encode eg what happens when it's 5mm exactly. 

On the principle of only asking for information we need for privacy, it's better to only ask if it's longer than 5mm - and it would also serve the principle of being transparent about the information we need for this particular decision.

But then we end up with another delegated decision tree.

A question for @verbman ...

Reply:
I'd suggest `beard.facial_hair_over_limit` - given that the 5mm parameter might change in the future.
Also you could ask `beard.facial_hair_length_in_mm > 5` which could be useful if the length of facial hair is something that other rules might be interested in.

Reply:
Have done!